_includes/open_graph.html

```diff
@@ -0,0 +1,25 @@
+<meta property="og:url" content="{{ site.url }}{{ page.url }}" />
+<meta property="og:type" content="article" />
+<meta property="og:title" content="{{ page.title | default: site.title }}" />
+{% if page.summary %}
+  {% assign description = page.summary %}
+{% elsif page.questions %}
+  {% assign description = page.questions | join: " " %}
+{% elsif page.keypoints %}
+  {% assign description = page.keypoints | join: " " %}
+{% elsif include.description %}
+  {% assign description = include.description %}
+{% else %}
+  {% assign description = "Learning guide for instructors and learners, with the opportunity for live coding examples." %}
+{% endif %}
+
+{% assign description = description | strip_html %}
+{% if description.size > 200 %}
+  {% assign description = description | truncate: 200 %}
+  {% assign description = description | append: "..." %}
+{% endif %}
+<meta property="og:description" content="{{ description }}" />
+<meta property="og:image" content="{{ site.url }}/assets/favicons/cp/mstile-310x310.png" />
+<meta property="og:image:type" content="image/png" />
+<meta property="og:image:alt" content="Carpentries' logo" />
```

Comment:
Modifying alt text if the logo belongs to The Carpentries or if it's one of the curriculum (software carpentry, data carpentry, etc) logos. 

```suggestion
{% if site.carpentry == "cp" %}
{% assign logo_alt = "The Carpentries' logo" %}
{% else %}
{% assign logo_alt = "The Carpentries' Curriculum logo" %}
{% endif %}
<meta property="og:image:alt" content="{{ logo_alt }}" />
```

Reply:
Do we have access to a `site.carpentry_full_name` or similar variable that we could use for more detail here, to produce e.g. "Data Carpentries Logo"?